source/layouts/landing.erb

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
```

Comment:
Add a `lang` attribute to improve accessibility by enabling technologies like screen readers to read out the text with correct pronunciation:

```suggestion
<html lang="en">
```

Even better, you can pull in the exact i18n locale, in case you do ever add another language:

```suggestion
<html lang="<%= I18n.locale %>">
```